Create ui-c8y-1019-7-7-adjust-widget-coordinates-and-size-to-fit-new-dashboard-grid.md (#1783)
SoftwareAG/c8y-docs: content/change-logs/ui-c8y-1019-7-7-adjust-widget-coordinates-and-size-to-fit-new-dashboard-grid.md

---
date: ""

> title: Adjust widget coordinates and size to fit new dashboard grid

Author: The Claude AI suggested to use the following title:
```suggestion
title: Improve widget placement on updated dashboard
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-3200
version: 1019.7.7
---

> Adjust widget coordinates and size to fit new dashboard grid

Author: The Claude AI suggested to use the following body:
```suggestion
The positions and sizes of widgets on the dashboard have been updated to fit with the new 12-column grid layout. This will allow for more precise control and consistency when placing multiple widgets on the dashboard. Any existing widgets may need to be adjusted to look correct on the new grid.
```

> In Device management, resetting the device "Info" dashboard resulted in widgets taking only half of the available dashboard space. Default dashboard settings have been now adjusted to fit new dashboard grid and "Reset dashboard" works properly again.

Reviewer: ```suggestion
Resetting the "Info" dashboard in the device details resulted in widgets taking only half of the available dashboard space. Default dashboard settings have now been adjusted to fit the new dashboard grid and resetting the dashboard works properly again.
```
